microservices-swift/Sources/Application/Routes/BoardgamesRoutes.swift

```diff
@@ -1,37 +1,41 @@
-import LoggerAPI
-import Health
+import Kitura
 import KituraContracts
 
-func initializeBoardgamesRoutes(app: App) {
-    let seed = [Boardgame(id: 1, name: "Risk", emoji: "💣"), Boardgame(id: 2, name: "Uno", emoji: "1️⃣")]
-    var boardgames: [Boardgame] = seed
+class BoardgameRoutes {
+    private static let seed = [Boardgame(id: 1, name: "Risk", emoji: "💣"), Boardgame(id: 2, name: "Uno", emoji: "1️⃣")]
+    private var boardgames: [Boardgame] = BoardgameRoutes.seed
     
-    app.router.get("/boardgames") { (respondWith: ([Boardgame]?, RequestError?) -> Void) -> Void in
-        respondWith(boardgames, nil)
-    }
-
-    app.router.post("/boardgames") { (boardgame: Boardgame, respondWith: (Boardgame?, RequestError?) -> Void) -> Void in
-        boardgames.append(boardgame)
+    public func initialise(with router: Router) {
+        let seed = [Boardgame(id: 1, name: "Risk", emoji: "💣"), Boardgame(id: 2, name: "Uno", emoji: "1️⃣")]
```

Comment:
Wait why having see as a static field and here seed has a block variable?

Reply:
good spot! removed the duplication!